tests/test_logsource.py

```diff
@@ -188,7 +188,7 @@ def test_fieldname_case(self):
                 list_valid = self.get_logsource(full_logsource)
                 fisrt_time = True
 
-                if list_valid != [] and self.not_commun(full_logsource,list_valid):
+                if list_valid != [] and list_valid != None and self.not_commun(full_logsource,list_valid):
```

Comment:
Both `[]` and `None` return `False` when evaluated as bools.

```suggestion
                if list_valid and self.not_commun(full_logsource,list_valid):
```